portfolio/src/main/webapp/script.js

```diff
@@ -38,6 +42,29 @@ function createMap() {
       });
 }
 
+/**
+ * Draws a chart and adds it to the page. Currently uses hardcoded data.
+ * TODO: get data from server.
```

Comment:
nit: Can you make your TODOs in the format 'TODO(shaargtz)'? That way if you're working on a codebase with multiple people (like during your capstone!) it'll be clear who is supposed to be doing what.

Reply:
Oh, alright, didn't know that was the way to handle it. Done!